_posts/2017-10-18-understanding-colourblindness.md

```diff
@@ -32,6 +32,6 @@ There are a number of online tools to help those without color blindness visuali
 
 - [Colorblind Color + Contrast Picker](http://colorsafe.co)
 - [Colorblind web page filter by Toptal](https://www.toptal.com/designers/colorfilter)
-- [I want to see like the colour blind Chrome extension](https://chrome.google.com/webstore/detail/i-want-to-see-like-the-co/jebeedfnielkcjlcokhiobodkjjpbjia)
+- [I want to see like the color blind Chrome extension](https://chrome.google.com/webstore/detail/i-want-to-see-like-the-co/jebeedfnielkcjlcokhiobodkjjpbjia)
```

Comment:
i noticed this one as well, but following the link the extension is actually titled "I want to see like the colour blind" so it probably should stay as the original spelling